### PR TITLE
Merge current v2.6 into master

### PR DESCRIPTION
**Pull request by tomMoulard**
### What does this PR do?

Merge current v2.6 branch into master.

### Motivation

Be synced.

### CODE REVIEW

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -26,7 +26,8 @@ global_job_config:
       - mkdir -vp "${SEMAPHORE_GIT_DIR}" "${GOPATH}/bin"
       - export GOPROXY=https://proxy.golang.org,direct
       - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "${GOPATH}/bin" v1.43.0
-      - curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | bash -s -- -b "${GOPATH}/bin"
+      - curl -sSfL https://gist.githubusercontent.com/traefiker/6d7ac019c11d011e4f131bb2cca8900e/raw/goreleaser.sh | bash -s -- -b "${GOPATH}/bin"
+      - go install github.com/containous/go-bindata/go-bindata@v1.0.0
```
Comment (ldez) [inline .semaphore/semaphore.yml:30]:
```suggestion
```

**Review by ldez — CHANGES_REQUESTED** [review]
it sounds like the v2.6 branch needs to be cleaned

File: .semaphore/semaphore.yml (hunk 1 above)
Comment (tomMoulard) [reply .semaphore/semaphore.yml:30]: fixed in #8667

**Review by ldez — APPROVED** [review]

**Pull request merged**